### ISSUE TITLE
Not load the signs

### ISSUE THREAD

**Issue by AlexandrALALA** [montlikadani/TeleportSigns#1]
```
[13:59:37] [Server thread/ERROR]: Error occurred while enabling TeleportSigns v1.3.7 (Is it up to date?)
java.lang.NullPointerException
	at org.bukkit.Location.getBlock(Location.java:87) ~[spigot.jar:git-Spigot-db6de12-18fbb24]
	at hu.montlikadani.TeleportSigns.ConfigData.loadSigns(ConfigData.java:159) ~[?:?]
	at hu.montlikadani.TeleportSigns.ConfigData.loadConfig(ConfigData.java:83) ~[?:?]
	at hu.montlikadani.TeleportSigns.TeleportSigns.onEnable(TeleportSigns.java:50) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321) ~[spigot.jar:git-Spigot-db6de12-18fbb24]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [spigot.jar:git-Spigot-db6de12-18fbb24]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot.jar:git-Spigot-db6de12-18fbb24]
	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.java:357) [spigot.jar:git-Spigot-db6de12-18fbb24]
	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.java:317) [spigot.jar:git-Spigot-db6de12-18fbb24]
	at net.minecraft.server.v1_8_R3.MinecraftServer.s(MinecraftServer.java:414) [spigot.jar:git-Spigot-db6de12-18fbb24]
	at net.minecraft.server.v1_8_R3.MinecraftServer.k(MinecraftServer.java:378) [spigot.jar:git-Spigot-db6de12-18fbb24]
	at net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:333) [spigot.jar:git-Spigot-db6de12-18fbb24]
	at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:263) [spigot.jar:git-Spigot-db6de12-18fbb24]
	at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:525) [spigot.jar:git-Spigot-db6de12-18fbb24]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_112]
```

**Comment by montlikadani** [comment]
Thanks for the mistake. Solved: https://www.spigotmc.org/resources/teleport-signs.37446/updates

**Comment by montlikadani** [comment]
Reopened this issue. This error still occurs.

**Comment by montlikadani** [comment]
**This issue has been fixed in v1.7** https://www.spigotmc.org/resources/teleport-signs.37446/update?update=224149
